handbook/about_dbt_labs.md

```diff
@@ -1,17 +1,28 @@
 # The dbt Labs Employee Handbook
 
+| Maintained by |
+|---|
+| Meg Colón |
+| Leigh Staub |
+
 ## Welcome to dbt Labs
-Hello and Welcome to dbt Labs. We’re thrilled to have you as a member of the team!
+Hello and welcome to dbt Labs. We’re thrilled to have you as a member of the team!
 
-This handbook is intended to answer many of the questions you might have after joining, and should serve as a valuable resource throughout your employment at dbt Labs. Read on to learn about important company policies and procedures, as well as the commitments of this company to our employees.
+This handbook is intended to answer your questions as a new employee and should serve as a valuable resource throughout your employment at dbt Labs. Read on to learn about important company policies and procedures, as well as the commitments of this company to our employees.
```

Comment:
"commitments of this company" sounds really corporate to me. Would we be open to "commitments [dbt Labs or we] make to our employees"